core/docz/src/components/Playground.tsx

```diff
@@ -14,38 +10,34 @@ export interface PlaygroundProps {
   __scope: Record<string, any>
   __position: number
   __code: string
-  language?: InternalPlaygroundProps['language']
+  language?: string
   showLivePreview?: boolean
 }
 
 export const Playground: SFC<PlaygroundProps> = ({
   className,
+  children,
   style,
-  wrapper: Wrapper,
+  wrapper,
   __scope,
   __position,
   __code,
   language,
-  showLivePreview: showLivePreviewFromUser = true,
+  showLivePreview = true,
 }) => {
-  const {
-    themeConfig: {
-      showPlaygroundEditor,
-      showLiveError,
-      showLivePreview = showLivePreviewFromUser,
-    },
-  } = useConfig()
+  const components = useComponents()
+  if (!components || !components.playground) return null
 
   return (
-    <DoczPlayground
+    <components.playground
```

Comment:
Maybe we can create a variable for this no? Makes it a bit clearer:

```
const components = useComponents()
const PlaygroundComponent = components.playground
```

Also the first `!components` is not necessary as there is already a default value here: https://github.com/jesperorb/docz/blob/6eda1289c872f891f58ac6e51c94d66938884294/core/docz/src/hooks/useComponents.tsx#L51